components/ui/Toast.tsx

```
@@ -0,0 +1,111 @@
+}
+
+/** Subscribe to toast state changes. */
+function subscribe(listener: () => void): () => void {
+  listeners.add(listener);
+  return () => listeners.delete(listener);
+}
+
+/** Show a toast message for 3 seconds. */
+function showToast(message: string) {
+  state = { message, visible: true };
+  listeners.forEach((l) => l());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)